src/Microsoft.Framework.PackageManager/SourceControl/SourceCommand.cs

```diff
@@ -0,0 +1,253 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Framework.Runtime;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NuGet;
+
+namespace Microsoft.Framework.PackageManager.SourceControl
+{
+    internal class SourceCommand
+    {
+        private const string SourcesFolderName = "sources";
+
+        private const string SourcesEnvironmentVariable = "DNX_SOURCES";
```

Comment:
I would think 3 renames are not hard you just replace all the `"DNX_` occurrences in the `EnvironmentNames.cs` file with Ctrl+Shift+H. The first time I almost missed that we use DNX_DEFAULTLIB in the managed code at all because I could not find a single occurrence of this string...